src/index.html

```diff
@@ -1,12 +1,393 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <div class="container">
+      <header class="header">
+        <a href="#Home">
+          <img
+            src="./images/logo.png"
+            alt="MoYo Logo"
+            class="header__logo"
+            >
+        </a>
+
+        <nav class="navigation">
+          <ul class="navigation__list">
+
+            <li class="navigation__item ">
+              <a href="#" class="navigation__link is-active">apple</a>
+            </li>
+
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">samsung</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">smartphones</a>
+            </li>
+            <li class="navigation__item" data-qa="nav-hover">
+              <a href="#" class="navigation__link">laptops & computers</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">gadgets</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">tablets</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">photo</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">video</a>
+            </li>
+
+          </ul>
+        </nav>
+      </header>
+
+      <main class="main">
+        <section class="catalog">
+          <article class="card" data-qa="card">
+            <a href="#" class="card__link">
+              <img
+                src="./images/imac.jpeg"
+                alt="Product image"
+                class="card__image-product">
```

Comment:
<img width="357" alt="attrCheklist" src="https://github.com/mate-academy/layout_catalog/assets/77551820/a21f43d3-f8af-4964-9365-147039f21b9f">

**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

Reply:
Not fixed